tgui/packages/tgui/styles/interfaces/PINInputModal.scss

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author ZeWaka (https://github.com/ZeWaka)
+ * @license MIT
+ */
+
+@import "../colors.scss";
+
+.PINInput {
+  display: flex;
+  justify-content: center;
+
+  .Button {
+    display: flex;
+    justify-content: center;
+    width: 3rem;
+    height: 3rem;
+    font-size: 1.7rem;
+
+    // Center icon buttons
+    .Icon {
+      margin-left: -0.1em;
```

Comment:
Hate this. Hate hate hate.

(Probably a necessary evil I guess for now.)

Reply:
how else would you do it

Reply:
Just silly that it's needed for horizontally aligning.